app/src/main/java/org/mozilla/fenix/settings/about/AboutFragment.kt

```diff
@@ -48,6 +50,8 @@ class AboutFragment : Fragment(), AboutPageListener {
     ): View? {
         val rootView = inflater.inflate(R.layout.fragment_about, container, false)
         appName = getString(R.string.app_name)
+        headerAppName =
+            if (Config.channel.isRelease) getString(R.string.daylight_app_name) else getString(R.string.app_name)
         activity?.title = getString(R.string.preferences_about, appName)
```

Comment:
We should probably check if the title for fragment should still be the `appName` - "About Firefox Preview". If not, we can probably just use either `appName` or `headerAppName`.

Reply:
Yeah I've pinged Michelle if we really _only_ want to replace the header here, because appName is used throughout the about page

Reply:
I looked at the original issue comments again. It doesn't look like we're checking the `appName` to "Firefox Daylight". I think you can disregard this comment.

Reply:
Confirmed with Michelle that it's just the header (not title, buttons etc)